package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A two-way JSON minifier to reduce JSON size and amount of data transferred on clients. Can also act as an obfuscator.",
   "main": "index.js",
   "scripts": {
-    "pretest": "jshint lib",
+    "pretest": "node_modules/.bin/jshint .",
```

Comment:
would `npm bin jshint .` also work?

Reply:
Didn’t know this command to be honest and you are right it’s even more cleaner and readable